osu.Game/Overlays/Chat/ChannelControl/ControlItem.cs

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics.Containers;
+using osu.Game.Online.Chat;
+
+namespace osu.Game.Overlays.Chat.ChannelControl
+{
+    public class ControlItem : OsuClickableContainer
+    {
+        public event Action<Channel>? OnRequestSelect;
+        public event Action<Channel>? OnRequestLeave;
+
+        public readonly BindableInt Mentions = new BindableInt();
+
+        public readonly BindableBool Unread = new BindableBool();
+
+        private readonly Channel channel;
+
+        private Box? hoverBox;
+        private Box? selectBox;
+        private ControlItemClose? close;
+
+        [Resolved]
+        private Bindable<Channel> selectedChannel { get; set; } = null!;
+
+        public ControlItem(Channel channel)
```

Comment:
Rename to `ChannelTab` or `ChannelListItem`, along with all the related classes.